### PR TITLE
feat: adding remaining python functions, removing unnecessary dependencies, standardizing env variables

### PR DESCRIPTION
**Pull request by hack3r-0m**
Relevant issues:
#19 #40

Testing:
tested `welcome-email` and `storage-cleaner` manually

### CODE REVIEW

**Comment by hack3r-0m** [comment]
@Glide7 can you please mark it DWOC?

File: python/welcome-email/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# 📧 Welcome Email 
+Sends the welcome email to the newly registered users to the app according to the template provided.
```
Comment (christyjacob4) [inline python/welcome-email/README.md:2]:
```suggestion
A sample Python Cloud Function for sending a welcome email to a newly registered user.
```

File: python/welcome-email/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## 📝 Environment Variables
+Go to Settings tab of your Cloud Function. Add the following environment variables.
+
+* **MAILGUN_API_KEY** =  API Key for Mailgun
+* **MAILGUN_DOMAIN** = Domain Name from Mailgun
```
Comment (christyjacob4) [inline python/welcome-email/README.md:8]:
```suggestion
* **MAILGUN_API_KEY** -  API Key for Mailgun
* **MAILGUN_DOMAIN** - Domain Name from Mailgun
```

File: python/welcome-email/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+* Upload the tarfile to your Appwrite Console and use the following entrypoint command
+
+```bash
+python main.py
+```
+
+* Navigate to the Overview Tab of your Cloud Function > Deploy Tag
+* Input the command that will run your function (in this case `python main.py`) as your entrypoint command
+* Upload your tarfile 
+* Click 'Activate'
```
Comment (christyjacob4) [inline python/welcome-email/README.md:44]:
```suggestion
* Navigate to the Overview Tab of your Cloud Function > Deploy Tag
* Input the command that will run your function (in this case `python main.py`) as your entrypoint command
* Upload your tarfile 
* Click 'Activate'
```

File: python/welcome-email/main.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+    return requests.post(
+        f"https://api.mailgun.net/v3/{os.environ['MAILGUN_DOMAIN']}/messages",
+            auth=("api", f"{os.environ['MAILGUN_API_KEY']}"),
+            data={"from": f"Excited User <mailgun@{os.environ['MAILGUN_DOMAIN']}>",
+                "to": [ email ],
+                "subject": f"Hello, {name}",
+                "text": "Testing some Mailgun awesomness!"})
+
+try:
```
Comment (christyjacob4) [inline python/welcome-email/main.py:22]: Have you tested that this function actually works ? and were you able to get an email sent to your inbox? 
If so, a screenshot would help

File: python/storage-cleaner/README.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
+# 🚮 Clean up files in your storage older than XX days
+A sample .NET Cloud Function for deleting files that are older than XX days on a schedule.
```
Comment (christyjacob4) [inline python/storage-cleaner/README.md:2]:
```suggestion
A sample python Cloud Function for deleting files that are older than XX days on a schedule.
```

File: python/storage-cleaner/README.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+* Upload the tarfile to your Appwrite Console and use the following entrypoint command
+
+```bash
+python main.py
+```
+
+* Navigate to the Overview Tab of your Cloud Function > Deploy Tag
+* Input the command that will run your function (in this case `python main.py`) as your entrypoint command
+* Upload your tarfile 
+* Click 'Activate'
```
Comment (christyjacob4) [inline python/storage-cleaner/README.md:46]:
```suggestion
* Navigate to the Overview Tab of your Cloud Function > Deploy Tag
* Input the command that will run your function (in this case `python main.py`) as your entrypoint command
* Upload your tarfile 
* Click 'Activate'
```

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Please address the comments

File: python/welcome-email/main.py (hunk 4 above)
Comment (hack3r-0m) [reply python/welcome-email/main.py:22]: image

Comment (hack3r-0m) [reply python/welcome-email/main.py:22]: image

Comment (christyjacob4) [reply python/welcome-email/main.py:22]: Perfect ❤️

**Review by christyjacob4 — APPROVED** [review]
Awesome Work!!!! 👍

**Comment by eldadfux** [comment]
🚀

**Pull request merged**